scalameter-core/src/main/scala/org/scalameter/package.scala

```diff
@@ -19,6 +19,18 @@ package object scalameter extends MeasureBuilder[Unit, Double](
   MeasureBuilder.average
 ) {
 
+  implicit def inline[T: Ordering](measurer: Measurer[T]): MeasureBuilder[Unit, T] = {
```

Comment:
Hm, it looks like signature should look like:

    def withMeasurer[V, M <: Measurer[V]](m: M, a: Seq[V] => V) // this M <: Measurer[V] is needed because Measurer is invariant and can't be made covariant

and there `test("Should correctly measure gc cycles") ` from `InlineBenchmarkTest` should look like this:
    
    withMeasurer(new Measurer.GarbageCollectionCycles, Aggregator.median[Int]) // manually specifying type to `Aggregator.median` is needed because otherwise wrong type is inferred

Or do you have other idea?